llm_on_ray/inference/inference_config.py

```diff
@@ -112,6 +112,27 @@ class ModelDescription(BaseModel):
     input_processor: str = "AutoProcessor"
     model_loader: str = "AutoModel"
 
+    chat_model_with_image: bool = False
+    chat_template: Union[str, None] = None
+    default_chat_template: str = (
+        "Below is an instruction that describes a task. Write a response that appropriately completes the request."
```

Comment:
This is like a system message, similarly, should it be part of the messages instead of in the template?